sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -281,13 +290,30 @@ export default defineComponent({
       applicationWizard = form;
     };
 
-    const pageChanged = (
+    const pageChanged = async (
       isInFirstPage: boolean,
       currentPage: number,
       isInLastPage: boolean,
     ) => {
       isFirstPage.value = isInFirstPage;
       isLastPage.value = isInLastPage;
+      if (!savedDraftData) {
+        savedDraftData = JSON.stringify(applicationWizard.submission.data);
+      }
+      const dataChanged = savedDraftData !== applicationWizard.submission.data;
```

Comment:
is it not suppose to be `const dataChanged =
        savedDraftData !== JSON.stringify(applicationWizard.submission.data);`?